syft/src/main/java/org/openmined/syft/execution/SyftJob.kt

```diff
@@ -66,7 +66,8 @@ class SyftJob internal constructor(
                 config,
                 JobRepository(
                     JobLocalDataSource(),
-                    JobRemoteDataSource(config.getDownloader())
+                    JobRemoteDataSource(config.getDownloader()),
+                    "$modelName-$version"
```

Comment:
JobId will later be also used for supplying request id parameters so we must ensure it is always unique. Better would be to create a class JobDataModel this will store the a unique 128bit hash as it's id, config, model name, model version. This datamodel will be fed to JobDataSource for usage.

Reply:
There is an internal JobID class, I guess with this approach we don't need it anymore